templates/blog/post_list.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html' %}
+
+{% block title %}
+  <title>Blog Posts</title>
+{% endblock %}
+
+{% block content %}
+  <h2>Posts</h2>
+  <div>
+    <ul>
+      {% for post in post_list %}
+        <li>
+          <h3><a href="{% url 'blog:post-detail' pk=post.id %}">{{ post.title }}</a></h3>
+          <div> Author: {{ post.owner }}</div>
+          <div>{{ post.created_time }}</div>
+          <div>{{ post.content }}</div>
```

Comment:
Consider limiting the length of the content displayed in the list of posts. You can use the Django template filter `truncatewords` to limit the number of words displayed. For example: `{{ post.content|truncatewords:30 }}`. This will display the first 30 words of the post content.